backend/node/genvm/base.py

```diff
@@ -13,6 +13,16 @@
 from backend.node.genvm.code_enforcement import code_enforcement_check
 
 
+class ContractRunner:
```

Comment:
Could you add comments explaining what each field represents please?

Reply:
Done!